.github/workflows/oc-pmrmrg.yml

```diff
@@ -0,0 +1,77 @@
+name: Github-CI Push/PR pmr_mrg
```

Comment:
I would rename the workflow to at least:
```Push/PR oc-pmr_mrg``` or ```Push/PR open-core-pmr_mrg```

Reply:
done